test/rpc_test.py

```diff
@@ -216,7 +217,7 @@ def test_self_add(self):
         with self.assertRaisesRegex(
             RuntimeError, "does not support making RPC calls to self"
         ):
-            rpc.rpc_sync(self_worker_info, torch.add, args=(torch.ones(2, 2), 1))
+            rpc.rpc_async(self_worker_info, torch.add, args=(torch.ones(2, 2), 1))
```

Comment:
Any reason for changing `rpc_sync` to `rpc_async`? Or do we actually want to test all three APIs (`rpc_sync`, `rpc_async`, and `remote`)?

Reply:
@mrshenli 

It looks to me this call and the following call are identical. I think the original purpose should be test both API?